plugins/services/src/js/pages/task-details/ServiceTaskDetailPage.js

```diff
@@ -16,11 +17,11 @@ class ServiceTaskDetailPage extends React.Component {
     const {id, taskID} = params;
 
     let routePrefix = `/services/overview/${encodeURIComponent(id)}/tasks/${encodeURIComponent(taskID)}`;
-    const tabs = [
+    const tabs = Hooks.applyFilter('serviceTaskDetailPageTabs', [
```

Comment:
The thing is that it goes like 2-3 levels deep so how'd you define a mounting point there? We would have to override all the nesting chain. Or am I missing something? Here I'd like to pass a ready to use set of elements instead of just configuration.